test/acceptance/docker-compose.yaml

```diff
@@ -0,0 +1,18 @@
+version: '3.8'
+
+services:
+  ffc-pay-batch-processor-acceptance-tests:
+    build:
+      context: .
+    image: ffc-pay-batch-processor-acceptance-tests
+    container_name: ffc-pay-batch-processor-acceptance-tests
+    environment:
+      AZURE_STORAGE_ACCOUNT_NAME: ${AZURE_STORAGE_ACCOUNT_NAME:-devstoreaccount1}
+      MESSAGE_QUEUE_HOST: ${MESSAGE_QUEUE_HOST}
+      MESSAGE_QUEUE_PASSWORD: ${MESSAGE_QUEUE_PASSWORD}
+      MESSAGE_QUEUE_USER: ${MESSAGE_QUEUE_USER}
+      PAYMENT_TOPIC_ADDRESS: ${PAYMENT_TOPIC_ADDRESS:-ffc-pay-request}${MESSAGE_QUEUE_SUFFIX}
+      PAYMENT_TOPIC_SUBSCRIPTION_ADDRESS: ${PAYMENT_TOPIC_SUBSCRIPTION_ADDRESS:-ffc-pay-enrichment}${MESSAGE_QUEUE_SUFFIX}
+      AZURE_STORAGE_USE_CONNECTION_STRING: "true"
+      AZURE_STORAGE_CONTAINERS: "batch"
+      AZURE_STORAGE_INBOUNDFOLDER: "inbound"
```

Comment:
What's the value in having a separate Dockerfile and Docker Compose set of files from the rest of the tests?  It just adds more complexity with docker network and volume setup as the orchestration required is no different from the other tests.

Suggest removing this folder's `Dockerfile`, `docker-compose.yaml`, `docker-compose.override.yaml`, `package-lock.json` and `package.json` files and instead using the existing resources to simplify the setup.

Any new dependencies can be added as `devDependencies` of the main file and the `test` script updated to support acceptance test running.  Then we inherit all behaviour of the docker file setup already without needing to repeat a separate set and deal with the complexity of bridging the two docker networks.